Makefile

```diff
@@ -211,4 +211,4 @@ ci-install-hypershift-private:
 		--oidc-storage-provider-s3-region=us-east-1 \
 		--private-platform=AWS \
 		--aws-private-creds=/etc/hypershift-pool-aws-credentials/credentials \
-		--aws-private-region=us-east-1
+		--aws-private-region=us-east-1
```

Comment:
unitended?

Reply:
Yes, Somehow it shows change to this file, Will remove this.